Lib/shutil.py

```diff
@@ -366,7 +376,7 @@ def lookup(name):
     _copyxattr(src, dst, follow_symlinks=follow)
 
 def copy(src, dst, *, follow_symlinks=True):
-    """Copy data and mode bits ("cp src dst"). Return the file's destination.
+    """Copy data and mode ("cp src dst"). Return the file's destination.
```

Comment:
I'm sorry, I didn't notice that the existing doc already used "mode bits". Please leave this doc unchanged ;-)